torchtext/data/datasets_utils.py

```diff
@@ -318,3 +319,26 @@ def pos(self):
 
     def __str__(self):
         return self.description
+
+
+@functional_datapipe("read_squad")
+class _ParseSQuADQAData(IterDataPipe):
```

Comment:
nit: we could move this to dataset_utils?  (as this one is also used for SQuaD2)

Reply:
That's where it already lives, right?

Reply:
Oops, my bad :). Thanks!